cookies/forms.py

```diff
@@ -279,3 +279,33 @@ class Meta:
 
 class ConceptEntityLinkForm(forms.Form):
     uri = forms.CharField(max_length=255, help_text='Manually enter a ConceptPower URI', label='URI')
+
+
+class UserAddCollectionForm(forms.Form):
```

Comment:
Can you add a docstring? I've been pretty bad at doing this myself, but moving forward we should always try to do this with functions and classes. 

Also, why not use a ``ModelForm`` here, instead of a ``Form``? If you use a ``ModelForm``, you don't need to implement the model instance creation logic in the view -- you'll get it for free!